tests/wpcom-vip-two-factor/sms-provider.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class Two_Factor_SMS_Test extends WP_UnitTestCase {
+	public function setUp() {
+		require_once __DIR__ . '/../../shared-plugins/two-factor/two-factor.php';
+		require_once __DIR__ . '/../../wpcom-vip-two-factor/sms-provider.php';
+	}
+
+	public function test__two_factor_sms_formatting() {
+		$two_factor = new Two_Factor_SMS();
```

Comment:
Because of the protected constructor (following the pattern from the `Two_Factor_Provider` abstract class/interface), we can't use the `new` keyword here. Instead we have to call `get_instance` 
```suggestion
		$two_factor = Two_Factor_SMS::get_instance();
```